packages/core/src/external/commonwell/cq-bridge/coverage-enhancer.ts

```diff
@@ -2,7 +2,7 @@ import { PatientLoader } from "../../../domain/patient/patient-loader";
 import { CQOrgHydrated, getOrgChunksFromPos, getOrgsByPrio, OrgPrio } from "./get-orgs";
 
 // Try to keep it even to make testing easier
-export const defaultMaxOrgsToProcess = 300;
+export const defaultMaxOrgsToProcess = 350;
```

Comment:
Increase since we're adding all high ones (currently 38 but can go up to 50 and we'll run an additional chunk anyways, so make the most out of it)